bridge/client/sdp.js

```diff
@@ -379,6 +387,11 @@ if (typeof(SDP) == "undefined")
         var sdpText = fillTemplate(templates.sdp, sdpObj);
         sdpText = fillTemplate(sdpText, sdpObj.originator);
 
+        if (!sdpObj.noTrickle)
+            var iceTrickleLine = "a=ice-options:trickle\r\n";
+
+        sdpText = fillTemplate(sdpText, { "trickleLine": iceTrickleLine });
```

Comment:
But msidSemanticLine is initialized to the empty string. My initial thought is that that's even necessary since the undefined iceTrickleLine will stringify to "undefined", but I may be wrong. So a ```var iceTrickleLine = ""``` before 388 and no "var" inside the if block would do it.